src/components/MapResult.tsx

```diff
@@ -3,14 +3,34 @@ import GoogleMapReact from 'google-map-react';
 
 // https://blog.logrocket.com/a-practical-guide-to-integrating-google-maps-in-react/
 
-const MapResult = ({ location, zoomLevel }) => {
+const location = {
+  // temporary for google maps, delete later
+  address: '1600 Amphitheatre Parkway, Mountain View, california.',
+  lat: 37.42216,
+  lng: -122.08427,
+}
+
+interface results {
+  cityName: string,
+  distance?: number,
+  travelTime?: number,
+  averageCost?: number
+}
+
+interface props {
+  results: results[],
+  statusCode: number,
+  message: string
```

Comment:
don't forget props contains lat, lng, and address of the searched city now. Also you can do:

`import {results, props} from "./result-body";
`
instead of initializing the interfaces locally

you can see what i did in request.tsx, i just commented the interfaces as a reference

Reply:
should be good besides that